### DISCUSSION TITLE
Dashboard inside Home assistant Core

### DISCUSSION THREAD

**Discussion by Julien-Decoen** [shannonhochkins/ha-component-kit#104] [Q&A]
Hello @shannonhochkins and thanks again for your work.

I'm running Home-Assistant Core and I would like to know if it's possible to run this dashboard inside Home-Assistant Core ?

Maybe tuning hakit ?

Thank you,
Julien

**Comment by shannonhochkins** [comment]
Hey mate! Have you tried it? I suspect it may not work, but I don't have a HA instance just running core, maybe give it a shot and report back?

**Reply by Julien-Decoen** [reply]
Hi there,

Sorry for the late. I tried but got a lot of refresh for the first display. 

Cannot got the time to go deeper... :/

**Reply by shannonhochkins** [reply]
I don't have anything setup with just home assistant core, you've not shown me any error messages at all, so it's very hard for me to help you.

What did you try? What happened?
